.travis.yml

```diff
@@ -25,7 +25,6 @@ script:
 - npm run lint
 - npm run test
 - npm run build
-- npm run is-es5
```

Comment:
We are explicitly leaving ES6 modules in to allow for tree shaking in a consuming app.